index.html

```diff
@@ -19,8 +19,176 @@ <h1>Sebastián<br>Muñoz</h1>
+                    </div>
+                    <div class="project-grid-item">
+                        <a href="" class="project-name">Project name goes here</a> 
+                    </div>
+                    <ul class="project-lang">
+                        <li>HTML/CSS</li>  
+                        <li>Ruby on Rails</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <div class="project-btn-grid btn">
+                        <button type="button" class="projects-btn">See this project <img src="assets/img/Union.png" alt=""></button>
+                    </div>
+                </div>
+            </li>
+            <li>
+                <div class="grid-container">
+                    <div class="picture">
+                        <img src="assets/img/Project5.png" alt="">
+                    </div>
+                    <div class="project-grid-item">
+                        <a href="" class="project-name">Project name goes here</a> 
+                    </div>
+                    <ul class="project-lang">
+                        <li>HTML/CSS</li>  
+                        <li>Ruby on Rails</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <div class="project-btn-grid btn">
+                        <button type="button" class="projects-btn">See this project <img src="assets/img/Union.png" alt=""></button>
+                    </div>
+                </div>
+            </li>
+            <li>
+                <div class="grid-container">
+                    <div class="picture">
+                        <img src="assets/img/Project6.png" alt="">
+                    </div>
+                    <div class="project-grid-item">
+                        <a href="" class="project-name">Project name goes here</a> 
+                    </div>
+                    <ul class="project-lang">
+                        <li>HTML/CSS</li>  
+                        <li>Ruby on Rails</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <div class="project-btn-grid btn">
+                        <button type="button" class="projects-btn">See this project <img src="assets/img/Union.png" alt=""></button>
+                    </div>
+                </div>
+            </li>
+        
+
+        </ul>
+    </section>
+    <section class="about-title">
+        <h1>About</h1>
+        <div class="intro">
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+        </div>
+        <div class="btn">
+            <button type="button">Get my CV  <img src="assets/img/ic_download.png" alt=""></button>
+        </div>
+        <div class="about-wrapper">
+            <ul class="lang">
+                <li>
```

Comment:
- [ ] This `li` tag is unnecessary. Kindly remove it. Do similar changes to the other categories too.